Frameworks/Foundation/NSObject.mm

```diff
@@ -705,11 +708,36 @@ void WinObjC_SetMissingSelectorFatal(BOOL fatal) {
 }
 @end
 
-    @implementation NSZombie
-    /**
-     @Status Interoperable
-    */
-    - (void)doesNotRecognizeSelector : (SEL)selector {
+static void _dealloc_zombify(id self, SEL _cmd); // forward declaration
+
+static void _NSCFZombifyHook(id object) {
+    _dealloc_zombify(object, nullptr);
+}
+
+@implementation NSZombie
++ (void)load {
+    __CFZombifyNSObjectHook = _NSCFZombifyHook;
+}
+
+// This implementation is required by CF, but NSZombie is a root class and does not inherit
+// NSObject's version.
++ (BOOL)isSubclassOfClass:(Class)classRef {
+    Class curClass = self;
+
+    while (curClass != nil) {
+        if (curClass == classRef) {
+            return YES;
+        }
+
+        curClass = class_getSuperclass(curClass);
+    }
```

Comment:
nope, just think it looks a bit cleaner. Shoulda been a nit, my b

---
In reply to: [81378408](https://github.com/Microsoft/WinObjC/pull/1059#discussion_r81378408) [](ancestors = 81378408)